src/Migrations/2023_06_23_100614_change_original_url_to_text.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    public function up(): void
+    {
+        Schema::table('laravel_ses_email_links', function (Blueprint $table) {
+            $table->text('original_url')->change();
+        });
+    }
+
+    public function down(): void
+    {
+        Schema::table('laravel_ses_sent_emails', function (Blueprint $table) {
```

Comment:
Hi @nicholaszuccarelli,
can you please fix the table name?

```suggestion
        Schema::table('laravel_ses_email_links', function (Blueprint $table) {
```